src/main/java/org/sagebionetworks/bridge/exporter/scheduler/BridgeExporterScheduler.java

```diff
@@ -0,0 +1,77 @@
+package org.sagebionetworks.bridge.exporter.scheduler;
+
+import java.io.IOException;
+
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.sqs.AmazonSQSClient;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+
+/** Bridge-EX 2.0 Scheduler */
+public class BridgeExporterScheduler {
+    private static final ObjectMapper JSON_OBJECT_MAPPER = new ObjectMapper();
+
+    private Table ddbExporterConfigTable;
+    private AmazonSQSClient sqsClient;
+
+    /**
+     * DDB table for Bridge Exporter Scheduler configs. We store configs in DDB instead of in env vars or in a config
+     * file because (1) DDB is easier to update for fast config changes and (2) AWS Lambda is a lightweight
+     * infrastructure without env vars or config files.
+     */
+    public final void setDdbExporterConfigTable(Table ddbExporterConfigTable) {
+        this.ddbExporterConfigTable = ddbExporterConfigTable;
+    }
+
+    /** SQS client, used to send the request to Bridge-EX. */
+    public final void setSqsClient(AmazonSQSClient sqsClient) {
```

Comment:
May not need to be public (Spring doesn't seem to need it). Then they don't show up in auto-complete, which is nice